dev/kyuubi-extension-spark-3-1/src/test/scala/org/apache/spark/sql/WatchDogSuite.scala

```diff
@@ -72,9 +72,9 @@ class WatchDogSuite extends KyuubiSparkSQLExtensionTest {
     withSQLConf(KyuubiSQLConf.WATCHDOG_FORCED_MAXOUTPUTROWS.key -> "10") {
 
       List("", "ORDER BY c1", "ORDER BY c2").foreach { sort =>
-        List("", " DISTINCT").foreach{ distinct =>
-        assert(sql(
-          s"""
+        List("", " DISTINCT").foreach { distinct =>
+          assert(sql(
+            s"""
```

Comment:
indent of L78-81

Reply:
+1, the indent here looks weird compare to others